run-basic-batch-operators-test.sh

```diff
@@ -49,6 +56,20 @@ OPERATOR_LIST_FILENAME=operator-list.txt
 # Operator list path in the report
 OPERATOR_LIST_PATH="$REPORT_FOLDER"/"$OPERATOR_LIST_FILENAME"
 
+# Log file
+LOG_FILENAME="output_$TIMESTAMP.log"
+
+# Operator list path in the report
+LOG_FILE_PATH="$REPORT_FOLDER"/"$LOG_FILENAME"
+
+echoColor() {
+	local color=$1
+	local message=$2
+
+	echo -e "${color}$message${ENDCOLOR}"
```

Comment:
Perhaps it would be more correct to send all messages to `stderr`?

Reply:
@edcdavid All diagnostic messages should go to `stderr` according to [this discussion](https://unix.stackexchange.com/questions/331611/do-progress-reports-logging-information-belong-on-stderr-or-stdout). It is debatable.

Reply:
I think that the way we currently have it complies with essence of the discussion, e.g. not to pollute the stdout output with diagnostics, that make sorting out the interesting info impossible. In our case we do not output diagnostics on the cli, logs are stored in a website for further retrieval. If we use stderr we would not be able to do this. Also by default there will be a lot of output that would pollute the cli by default.